frontend/src/pages/admin/TeamTypes/dialogs/TeamTypeEditDialog.vue

```diff
@@ -163,11 +164,14 @@ export default {
                     if (this.input.properties.features.customCatalogs === undefined) {
                         this.input.properties.features.customCatalogs = true
                     }
+                    if (this.input.properties.features.emailAlerts === undefined) {
+                        this.input.properties.features.emailAlerts = true
```

Comment:
```suggestion
                        this.input.properties.features.emailAlerts = false
```

Default should be disabled until they are added to a TeamType.